app/build.gradle

```diff
@@ -62,7 +62,7 @@ dependencies {
     // the maven declaration of Maps Compose can be used as a snippet.
     // implementation project(':maps-compose')
     // [END_EXCLUDE]
-    implementation  "com.google.maps.android:maps-compose:2.1.1"
+    implementation  "com.google.maps.android:maps-compose:2.2.0"
```

Comment:
Looks like 2.2.0 isn't published yet. FYI @barbeau @wangela

Reply:
Thanks @arriolac, it should now be available:
https://search.maven.org/artifact/com.google.maps.android/maps-compose

Reply:
Thanks @arriolac, it should now be available:
https://search.maven.org/artifact/com.google.maps.android/maps-compose